keyvalue/examples/src/main/java/tachyon/examples/keyvalue/hadoop/CloneKeyValueStore.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.examples.keyvalue.hadoop;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.BytesWritable;
+import org.apache.hadoop.mapred.FileInputFormat;
+import org.apache.hadoop.mapred.FileOutputFormat;
+import org.apache.hadoop.mapred.JobClient;
+import org.apache.hadoop.mapred.JobConf;
+import org.apache.hadoop.mapred.MapReduceBase;
+import org.apache.hadoop.mapred.Mapper;
+import org.apache.hadoop.mapred.OutputCollector;
+import org.apache.hadoop.mapred.Reducer;
+import org.apache.hadoop.mapred.Reporter;
+
+import tachyon.client.keyvalue.hadoop.KeyValueInputFormat;
+import tachyon.client.keyvalue.hadoop.KeyValueOutputCommitter;
+import tachyon.client.keyvalue.hadoop.KeyValueOutputFormat;
+
+/**
+ * This boring MapReduce job clones a key-value store to a different URI.
+ */
+public class CloneKeyValueStore {
```

Comment:
mark this class final?
also, rename this to `CloneKeyValueStoreMapReduce` maybe? In that way, it is explicit from class/file name that this is an example of MapReduce using keyvalue API